include/NovelRT/Audio/Audio.h

```diff
@@ -0,0 +1,33 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_AUDIO_H
+#define NOVELRT_AUDIO_H
+
+#include "../LoggingService.h"
+#include "../Utilities/Lazy.h"
+#include <AL/al.h>
+#include <AL/alc.h>
+#include <sndfile.h>
+//#include "../Maths/GeoVector2F.h"
+
+namespace NovelRT::Audio
+{
+    class AudioService;
+}
```

Comment:
This looks to be living outside of the C++ check? I'm not sure I 100% understand what's happening here.

Reply:
This appears to be what we are doing with `Ecs.h`, `Graphics.h`, `Windowing.h`, etc.. so I was just following the semantic there.

Reply:
`Ecs.h` has no C API calls. The header is excluded via `NovelRT.h` if the C API is used, I'm fairly sure? I might be misunderstanding something here though. This sounds like a possible bug vector otherwise.

Reply:
Odd, I thought we had this for a reason (it was in NovelRT.h at some point im pretty sure). I've removed it in a new commit, compiles fine ¯\_(ツ)_/¯